src/components/service-events/controllers.test.tsx

```diff
@@ -218,8 +225,8 @@ describe('service events', () => {
 
       expect(response.body).toContain('Displaying page 1 of 2702');
       expect(response.body).toContain('1337 total events');
-      expect(response.body).toContain('<a class="govuk-link" disabled>Previous page</a>');
-      expect(response.body).not.toContain('<a class="govuk-link" disabled>Next page</a>');
+      expect(response.body).toContain('<a class=\"govuk-link\">Previous page</a>');
+      expect(response.body).not.toContain('<a class="govuk-link">Next page</a>');
```

Comment:
If the tests passed I guess it doesn't matter these two lines inconsistently escape double quotes 🤷‍♀